browser/resources/settings/brave_leo_assistant_page/brave_leo_assistant_page.ts

```diff
@@ -70,6 +108,21 @@ class BraveLeoAssistantPageElement extends BraveLeoAssistantPageBase {
       }
     }
 
+    toModelName_(modelKey: string) {
+      return MODEL_NAMES.get(modelKey)
+    }
+
+    // TODO(nullhook): share these strings with AIChat webui
+    getModelSubtitleLocale_(modelKey: string) {
```

Comment:
function is not returning a locale, so it can be named `getModelSubtitle`

Reply:
done